clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/main/views/skins/SaveButtonSkin.mxml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<s:Skin xmlns:fx="http://ns.adobe.com/mxml/2009"
+		xmlns:s="library://ns.adobe.com/flex/spark">
+	<!-- host component -->
+	<fx:Metadata>
+		[HostComponent("spark.components.Button")]
+	</fx:Metadata>
+
+	<!-- states -->
+	<s:states>
+		<s:State name="disabled" />
+		<s:State name="down" />
+		<s:State name="over" />
+		<s:State name="up" />
+	</s:states>
+	<s:Label id="labelDisplay"
```

Comment:
The `color` is applied from `.saveButton` css class and the font size is read from the css resolution file; `xhdpi.css` in our case. The button class is in charge of reading them.